File: GTTS/INL.TrainingService.Models/AttachDocumentToTrainingEventParticipant_Item.cs
using System;

namespace INL.TrainingService.Models
{
    public class AttachDocumentToTrainingEventParticipant_Item : IAttachDocumentToTrainingEventParticipant_Item
    {
        public long TrainingEventStudentAttachmentID { get; set; }
        public long TrainingEventID { get; set; }
        public long PersonID { get; set; }
        public int FileVersion { get; set; }
        public int TrainingEventStudentAttachmentTypeID { get; set; }
        public string TrainingEventStudentAttachmentType { get; set; }
        public string FileName { get; set; }
        public string Description { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public DateTime ModifiedDate { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/attach-document-to-training-event-participant_item.ts


import { IAttachDocumentToTrainingEventParticipant_Item } from './iattach-document-to-training-event-participant_item';

export class AttachDocumentToTrainingEventParticipant_Item implements IAttachDocumentToTrainingEventParticipant_Item {
  
	public TrainingEventStudentAttachmentID: number = 0;
	public TrainingEventID: number = 0;
	public PersonID: number = 0;
	public FileVersion: number = 0;
	public TrainingEventStudentAttachmentTypeID: number = 0;
	public TrainingEventStudentAttachmentType: string = "";
	public FileName: string = "";
	public Description: string = "";
	public ModifiedByAppUserID: number = 0;
	public ModifiedDate: Date = new Date(0);
  
}






